docs/manual/java/guide/production/index.toc

```diff
@@ -1,3 +1,3 @@
 ProductionOverview:Production Overview
-EnterpriseSuite:Lightbend Enterprise Suite
```

Comment:
Unfortunately this rename means that any links to the previous URL will return 404. I couldn't figure out a way to redirect. Does anyone know?

Reply:
If there isn't a way to redirect, I guess we have three options:

1. Live with the 404 (current approach in this pull request)
2. Keep `EnterpriseSuite.html` in the URL and only change the contents of the page
3. Change play-doc/Lightbend Markdown to add a way to generate static redirect pages (lagom.github.io has some generated pages that redirect using JavaScript, but only in the top level of the site, not in the versioned documentation)

Reply:
I think there is another workaround. We can add a task on sbt that after generating the site, adds a .html file called `EnterpriseSuite.html` with a static redirect. It's hacky, but may be the simplest solution for now.

Reply:
We can also keep `EnterpriseSuite.html` page but with text explaining there is now Lightbend Platform and have a link to the new page.

Reply:
I should have mentioned that I tried @marcospereira's suggestion already: leaving a file named `EnterpriseSuite.md` with a static link to `LightbendPlatform`. The problem is that play-doc doesn't even look at pages unless they're listed in the TOC, and I didn't think we should do that.